wix/Build.OpenJDK_generic.cmd

```diff
@@ -194,10 +201,9 @@ FOR %%A IN (%ARCH%) DO (
 
 
     REM Build with extra Source Code feature (needs work)
```

Comment:
I updated the code in the comments below to be in the style of wix v4 and wix v5, but it seems that these comments have been untouched for a while now. Would it be better to delete them?